MekHQ/src/mekhq/gui/CampaignGUI.java

```diff
@@ -900,7 +911,7 @@ private void initMenu() {
         // The Reports menu uses the following Mnemonic keys as of 19-March-2020:
         // C, H, P, T, U
         JMenu menuReports = new JMenu(resourceMap.getString("menuReports.text")); // NOI18N
-        menuReports.setMnemonic(KeyEvent.VK_R);
+        menuReports.setMnemonic(KeyEvent.VK_E);
```

Comment:
Why would this not be `R`?

Reply:
Ran out of mnemonic letters, and because of that I prioritized based on use. Repair tab sees more use than this one, so that's why.